server/controllers/message.js

```diff
@@ -0,0 +1,33 @@
+const Message = require("../models/Message");
+const asyncHandler = require("express-async-handler");
+
+
+//Based on ConversationID and SenderID, Create a message
+exports.createMessage = asyncHandler(async (req, res) => {
+    const { senderID, convoID, message } = req.body;
+    const newMessage = new Message({
+        conversationID: convoID,
+        senderID,
+        message,
+    });
+
+    try {
+        const savedMessage = await newMessage.save();
+        res.status(200).json(savedMessage);
```

Comment:
Please use appropriate status code for resource creation. 

Ref: https://www.restapitutorial.com/httpstatuscodes.html